components/chainhook-cli/src/service/mod.rs

```diff
@@ -38,6 +39,7 @@ impl Service {
     pub async fn run(
         &mut self,
         predicates_from_startup: Vec<ChainhookFullSpecification>,
+        observer_commands_tx_rx: Option<(Sender<ObserverCommand>, Receiver<ObserverCommand>)>,
```

Comment:
Being able to create the observer commands tx externally allows stopping the service from a test.